conans/test/integration/toolchains/cmake/cmakedeps/test_cmakedeps.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+from conans.test.assets.genconanfile import GenConanfile
+from conans.test.utils.tools import TestClient
+
+
+def test_package_from_system():
+    """
+    If a node declares "system_package" property, the CMakeDeps generator will skip generating
+    the -config.cmake and the other files for that node but will keep the "find_dependency" for
+    the nodes depending on it. That will cause that cmake looks for the config files elsewhere
+    https://github.com/conan-io/conan/issues/8919"""
+    client = TestClient()
+    dep2 = str(GenConanfile().with_name("dep2").with_version("1.0")
+               .with_settings("os", "arch", "build_type", "compiler"))
+    dep2 += """
+    def package_info(self):
+        self.cpp_info.set_property("external_package", True)
```

Comment:
I'd probably try to make it more explicit like ``skip_deps_file``, or ``external_build_script``, or  something like that, because "external package" could be a system package, but you still want the xxx-config.cmake or equivalent created to work.

But I guess mostly a personal taste and naming opinion, so I am fine with it, you can merge it as-is.

Reply:
I was waiting for you name suggestion. `skip_deps_file` will be